lib/active_versioning/model/version_proxy.rb

```diff
@@ -99,7 +99,7 @@ def commit(params = {})
 
         version.update(attrs)
 
-        __getobj__.update(versioned_attributes)
+        version.versionable.update(versioned_attributes)
```

Comment:
What I wanted to do here was reload the underlying model before running the update, because if I don't, new associated models get created twice (one gets built when the model is loaded up, the other comes in through versioned_attributes). Couldn't figure out how to reload (`__getobj__.reload` didn't work), but just reaching into the version seems to work just fine.